### DISCUSSION TITLE
Your respiratory  under different profile

### DISCUSSION THREAD

**Discussion by Boka-Krista** [community/community#134166] [New to GitHub]
Has anyone come across that you find you own respiratory in someone elses profile while you don't even know the person?
So I was working on a website to practice the coding skills only just to find (by accident) that the whole thing (with few missing code bits) is also in somebodys profile? Hope that makes sense as at the moment i'm too confused as of why and how?
![git](https://github.com/user-attachments/assets/60565bf7-6354-4f2d-ac6d-4f4d53a3a6af)

**Comment by Buddhi19** [comment]
Might be a fork from your repo

**Reply by Boka-Krista** [reply]
But I’m still wondering if it’s still kinda allowed? As I’ve been working on it for couple of months and profile currently for personal use only

**Reply by Buddhi19** [reply]
The image you have added here shows that the final commit is done by you so it's clear that it is a fork from your repo.
If you have set your repo private you might have added this person as a collaborator (still have the option to fork).
If that's not the case I think it’s better you change your authentication stuff (pw etc)

**Reply by Boka-Krista** [reply]
Then could have happened before I deleted it maybe because at the moment it shows 0 forks. And noticed this literally just yesterday 🙈

**Comment by ghostinhershell** [comment]
Hi @Boka-Krista, 

Thanks so much for reaching out in our community. From your screenshot, I do see that the option to fork your repository is present, although it doesn't look like your repo has actually been forked based on the count that's next to it. Is it possible that it's a coincidence that the other person's repo is similar?

For more information on how you can disable the ability for other's to fork your repo, check out our support article here: [Managing the forking policy for your repository](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-the-forking-policy-for-your-repository).

**Reply by Boka-Krista** [reply]
It’s too much of a coincidence. Because all the files and content is 1:1 apart of search keywords in JavaScript file. https://github.com/Boka-Krista/natural-remedies Here’s the link to mine which I redone for said page (as don’t know why the translate option didn’t go away after deleting that part of code, so placed the code in “new” respiratory and the one in screenshot I was deleting

**Comment by nirali393** [comment]
You can check the timeline of the other person’s project or reach out to the person directly

**Reply by Boka-Krista** [reply]
Last updates are there from about last month, which lines up when I copied the files to new one and deleted that one. Also just noticed that the readme file is now empty (while it should have been filled out) but not sure on how to contact that person directly as I thought about it, but ended up asking here
